debify.gemspec

```diff
@@ -19,19 +19,18 @@ Gem::Specification.new do |spec|
   
   spec.add_dependency "gli"
   spec.add_dependency "docker-api", "~> 2.0"
-  spec.add_dependency "conjur-cli" , "~> 6"
-  spec.add_dependency "conjur-api", "~> 5"
-
+  #spec.add_dependency "conjur-cli" , "~> 6"
+  spec.add_dependency "conjur-api", "~> 5.pre", ">= 5.3.6"
   spec.add_development_dependency "bundler", ">= 2.2.30"
   spec.add_development_dependency "fakefs", "~> 0"
-  spec.add_development_dependency "rake", "~> 13.0"
+  spec.add_development_dependency "rake", "~> 13.0.6"
   
   # Pin to cucumbe v2. cucumber v3 changes (breaks) the behavior of
   # unmatched capture groups with \(d+). In v3, the value of such a
   # group is 0 instead of nil, which breaks aruba's "I successfully
   # run...." steps.
-  spec.add_development_dependency "cucumber", '~> 2'
-  spec.add_development_dependency "aruba", "~> 1.0"
-  spec.add_development_dependency 'rspec', '~> 3'
-  spec.add_development_dependency 'ci_reporter_rspec', '~> 1.0'
+  spec.add_development_dependency "cucumber", '~> 7.1.0'
+  spec.add_development_dependency "aruba", "~> 2.0"
+  spec.add_development_dependency 'rspec', '~> 3.10.0'
+  spec.add_development_dependency 'ci_reporter_rspec', '~> 1.0.0'
```

Comment:
Why are you using 3 components in version

Reply:
Fixed